src/app/settings/options.component.ts

```diff
@@ -80,6 +90,16 @@ export class OptionsComponent implements OnInit {
         await this.stateService.save('enableGravatars', this.enableGravatars);
         await this.storageService.save('enableFullWidth', this.enableFullWidth);
         this.messagingService.send('setFullWidth');
+        if (this.theme !== this.startingTheme) {
+            this.changeTheme = this.startingTheme = this.theme;
```

Comment:
Unless I'm overlooking something, `changeTheme` doesn't need to be a property - it can just be a local variable declared with `let`. 

Also, maybe `effectiveTheme` would be a clearer name? (It's kind of difficult to distinguish between what you've chosen vs. what actual theme to apply)